tests/assets/config.json

```diff
@@ -0,0 +1,15 @@
+
+{
+    "HB_layers" :{
```

Comment:
Why do you need an extra key here? Can't we just have a flattened structure if you are only using this file for layers and remove the "HB_layers" key?

```
{       "HB_wall": "",
        "HB_grid": "grid",
        "HB_room": "",
        "HB_floor": " ",
        "HB_roof": " ",
        "HB_airwall": " ",
        "HB_view": " ",
        "HB_aperture": "window",
        "HB_shade": "shade",
        "HB_door": "door"
    }
```